src/pages/player-info.md

```diff
@@ -0,0 +1,91 @@
+---
+title: Information For Players
+subtitle: >-
+image: 
+template: page
+---
+# Overview
+For those who aren’t familiar, [Jackbox Party Games](https://www.jackboxgames.com/) are a set of quirky, well, party games. They consist of fun takes on Pictionary, Balderdash, Trivial Pursuit, and more. In the world of isolation, Jackbox games have been used via Zoom and other platforms to play with friends and family members around the world.
+
+DevOps Party Games takes this idea and tilts it on its head by adding DevOps-inspired content to the Jackbox games [Drawful 2](https://www.jackboxgames.com/drawful-two/) and [Quiplash 2](https://www.jackboxgames.com/quiplash-two-interlashional/), and then streams it live via [Twitch](https://twitch.tv/mattstratton) for a worldwide audience to watch, comment, and hopefully be entertained. In addition, the hosts ([Matt Stratton](https://twitter.com/mattstratton), [Jeremy Meiss](https://twitter.com/IAmJerdog), and [Dan Maher](https://twitter.com/phrawzty)) will provide color commentary, much like a modern day Cotton McKnight and Pepper Brooks (announcers from Dodgeball). The goal here is to have fun and not take things too seriously.
+
+If you’d like to see an example, previous episodes can be viewed at [http://matty.wtf/devops-party-games](http://matty.wtf/devops-party-games).
+
+# The Format
+There will be two rounds - one round of Quiplash, and one round of Drawful.
+
+There will be a maximum of 8 contestants, who will play both rounds.
+
+We will be using “custom episodes” of Drawful and Quiplash, where all the prompts will be related to devops/cloud computing/etc. 
+
+So for example, a prompt you might get in Drawful is that you need to draw a picture of “full stack engineer” and the other players have to try to guess that!
+
+## Code of Conduct
+All players will need to abide by a [Code of Conduct](https://devopspartygames.com/code-of-conduct). Remember that this will be broadcast live via Twitch, and also will be recorded to be viewed later, so keep that in mind!
+
+Unofficially, the ESRB rating for this will be “M” (Mature, 17+)
+
+That said, please refrain from sexualized jokes/content in gameplay.
+
+## Logistics
+All contestants and hosts will connect to Zoom video call (the Zoom link will be in your calendar invite). Your video will be part of the stream—if you would prefer to be “camera off” please just let us know.
+
+Players will see the “screen” of the game via Zoom screen sharing. They will use a second device (tablet, smartphone, etc) as the “controller” for the game (the place where they provide input/draw pictures, etc).
```

Comment:
```suggestion
Players will see the “screen” of the game via Zoom screen sharing. They will use a second device (tablet, smartphone, etc) as the “controller” for the game (the place where they provide input/draw pictures, etc). Players with multiple displays (or one _really big_ one) can use the same screen for both.
```

Reply:
yes, but we might want to remind them that the drawing games can be hard on a computer vs mobile device?